test/testcafe/spec/AuthenticatorIdvView_spec.js

```diff
@@ -38,7 +38,10 @@ test
     await checkA11y(t);
     console.log(pageObject.getFormTitle());
     await t.expect(pageObject.getFormTitle()).eql('Verify your identity with Persona');
-    await t.expect(pageObject.getPageSubtitle()).eql('Verify your identity before setting up account to ensure that only the real you has access.');
+    await t.expect(pageObject.getPageSubtitle()).eql('This strengthens security by validating your identity with official documentation.');
+    if(!userVariables.gen3) {
+      await t.expect(pageObject.getBeaconSelector()).contains('mfa-idv-persona');
+    }
```

Comment:
nit: Could you add a small comment here that explains it was a design decision to _not_ have a logo/beacon for Gen 3?